crates/orderbook/src/arguments.rs

```diff
@@ -146,6 +155,11 @@ impl std::fmt::Display for Arguments {
             "max_order_validity_period: {:?}",
             self.max_order_validity_period
         )?;
+        writeln!(
+            f,
+            "max_limit_order_validity_period: {:?}",
+            self.max_limit_order_validity_period
+        )?;
```

Comment:
This implements Display so Default is unrelated but you probably mean Default. We do it because we don't want to print some arguments through their Debug implementation. For example urls become many lines and some secrets shouldn't be printed at all. There is a PR by me that switched from derive Debug to manual Display impl where I made a case for it.